app/src/main/java/com/github/geohunt/app/ui/components/tutorial/TutorialSlides.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.geohunt.app.ui.components.tutorial
+
+import com.github.geohunt.app.R
+
+/**
+ * The data class for the tutorial slides
+ *
+ * @param icon The icon displayed on the slide
+ * @param title The title of the slide
+ * @param description The description for the slide
+ */
+class TutorialSlides(
```

Comment:
It might be clearer to transform this class into an enum (and you could then access to Java's standard .values() method)

Reply:
I disagree with your suggestion for two reasons:
1) It's preferable to use Kotlin methods whenever possible.
2) The class in its current form maintains high code lisibility and maintainability. I do not think that changing the class to ENUM would make it any clearer or better.